groups/templates/groups/group_subscribe_button.html

```diff
@@ -0,0 +1,5 @@
+{% load crispy_forms_tags %}
+
+<div id="group-subscribe-button">
+    {% crispy subscribe-form %}
```

Comment:
This will clash with the other `subscribe-form` that might be in the context. One of them will take precedence, and override the other. Please use another variable name.